city_scrapers/spiders/chi_ssa_22.py

```diff
@@ -0,0 +1,124 @@
+# import logging
+import unicodedata
+from datetime import datetime
+
+import dateutil.parser
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa22Spider(CityScrapersSpider):
+    name = "chi_ssa_22"
+    agency = "Chicago Special Service Area #22 Clark St/Andersonville"
+    timezone = "America/Chicago"
+    start_urls = ["http://www.andersonville.org/our-organizations/andersonville-ssa-22/"]
+    location = {
+        "name": "Andersonville Chamber of Commerce",
+        "address": "5153 N. Clark St. #228 Chicago, Illinois 60640"
```

Comment:
Small thing, but we've been using the state abbreviation instead of the name so Illinois should be IL

Reply:
Still that is of course important. thanks for pointing it out.